src/main/scripts/src/app/core/workspace.service.ts

```diff
@@ -26,6 +26,9 @@ export class WorkspaceService {
   public organizationCode: string;
   public projectCode: string;
 
+  public actualOrganizationCode: string;
+  public actualProjectCode: string;
```

Comment:
Isn't there a better name for the variables? I am not sure what's not actual about the `organizationCode` and `projectCode` codes.

Reply:
The difference is that the `organizationCode` and `projectCode` variables are only changed when both are changed together. But maybe there is a better name for them or a better way how to do it. Any suggestions?

Reply:
I am thinking of some: 
  - Name them `currentlySelectedOrganizationCode` and `currentlySelectedProjectCode`
  - If the workspace is chosen organization, project pair.. you could remove them from the service and pass the `organizationCode` to the `OrganizationService` methods.